src/ucp/core/ucp_context.h

```diff
@@ -207,6 +207,12 @@ typedef struct ucp_context {
     /* Mask of memory type communication resources */
     uint64_t                      mem_type_access_tls[UCS_MEMORY_TYPE_LAST];
 
+    struct {
+        ucs_time_t                timeout;    /* Time period between keepalive rounds */
+        unsigned                  num_eps;    /* Maximal number of endpoints to check
+                                               * on every round */
+    } keepalive;
```

Comment:
plz move to non external config, like with other vars

Reply:
moved